File: prog4.c
#include<stdio.h>

int main()
{
    char check;
    printf("Enter the character: ");
    scanf("%c",&check);

    if (check=='a'||check=='A'||check=='e'||check=='E'||check=='i'||check=='I'||check=='o'||check=='O'||check=='u'||check=='U')
    {
        printf("The character %c is a vowel",check);
    }
    else
    {
        printf("The character %c is a consonant",check);
    }
    
    return 0;
}

File: prog4.py
char=input("Enter the character: ")

char_str=[]
for i in char:
    char_str.append(i)

if len(char_str)>1:
    print(f'{char} is not supported')
elif char_str[0] in 'aeiou':
    print(f'{char} is a vowel')
else:
    print(f'{char} is a consonant')